misc/kinetics_400_labels.csv

```diff
@@ -0,0 +1,401 @@
+id,name
```

Comment:
Could we move it under demo/

Reply:
i will change that and the default config value as well